docs/source/architecture/injecting_batches_block_validation_rules.rst

```diff
@@ -23,7 +23,7 @@ some use cases.  Two examples of inter-transaction validation are:
 BatchInjector Interface
 =======================
 
-The ``BatchInjector`` class supports injecting transactions into blocks:
+The ``BatchInjector`` class supports injecting batches into blocks:
```

Comment:
This section would require revision as well. Here's the latest code https://github.com/hyperledger/sawtooth-core/blob/master/validator/sawtooth_validator/journal/batch_injector.py and the comment on block_start is changed.